shared/fs/filepreview/default-view.js

```diff
@@ -77,15 +82,17 @@ const DefaultView = (props: DefaultViewProps) => (
         onClick={props.onDownload}
       />
     )}
-    {props.pathItem.name.indexOf('.') === -1 && (
-      <Button
-        key="open-text"
-        type="Secondary"
-        label="Open as text"
-        style={{marginTop: globalMargins.small}}
-        onClick={props.onOpenAsText}
-      />
-    )}
+    {props.pathItem.name.indexOf('.') === -1 &&
```

Comment:
remind me, what is the purpose of this check?

Reply:
This is so that we only show this button when a file doesn't have an extension.

Reply:
Hmm, why?

Reply:
I was gonna do this for all unknown files, but Cecile convinced me we should only do it for files with no extension, as we don't want to open a e.g. Illustrator file as plain text.

Reply:
Sounds good, thanks. An explanatory comment in the code would be helpful :)

Reply:
Sure! I'll try to get this merged and add it in my next PR.